.github/ISSUE_TEMPLATE/bug-report--tiberiumrim-mod-.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report (TiberiumRim Mod)
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### **Describe the bug**
+  <!--- This should be the last thing you fill out. Start with the other steps first and come back here after that! -->
+A clear and concise description of what the bug is.
+
+### **Describe your game setup**
+  <!--- This can help us a lot! Please pick/replace whatever you see fit most! -->
+1. - [ ] I added TiberiumRim to an older, already created colony 
+   - [ ] I installed TiberiumRim and then created a fresh colony
+2. - [ ] Runnnig **only TiberiumRim Mod (and HugsLib + Harmony)** 
```

Comment:
Hugslib is not necessary, only Harmony as a dependency.
Hugslib is useful for log-sharing though, should be recommended separetly.